docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -13,6 +13,21 @@ Each component in a single devfile must have a unique name and use one of the ob
 |componentObject
 |no
 |The list of containers that you use.
```

Comment:
```suggestion
|The list of containers
```
You don't lose any meaning by deleting `that you use.` from all of these entries